system_files/silverblue/usr/share/glib-2.0/schemas/zz0-bluefin-modifications.gschema.override

```diff
@@ -77,7 +75,7 @@ sort-directories-first=true
 sort-directories-first=true
 
 [org.gnome.mutter]
-experimental-features=['scale-monitor-framebuffer']
+experimental-features=['scale-monitor-framebuffer', 'xwayland-native-scaling']
```

Comment:
Does this value cause an issue on Gnome 40?

Reply:
Not that I could tell.  It just sits unused